app/constants/riff_metting.js

```diff
@@ -4,6 +4,7 @@ export default {
     WEBVIEW_HEADERS: {
         'X-Mobile-App': 'mattermost',
     },
+    WEBVIEW_META_VIEWPORT: 'const meta = document.createElement(\'meta\'); meta.setAttribute(\'content\', \'width=device-width, initial-scale=1\'); meta.setAttribute(\'name\', \'viewport\'); document.getElementsByTagName(\'head\')[0].appendChild(meta); ',
```

Comment:
You can use backticks to define this constant so that you don't have to escape single quotes inside the string by using `\'`.

Reply:
fixed [here](https://github.com/Brightscout/riff-mattermost-mobile-automation/pull/10/commits/3d26498d64541e8822384d48272a8b77fb22b6d9)